### PR TITLE
Feature/2.0.8.3 quests

### PR DESCRIPTION
**Pull request by richardhendricks**
Ready for release 2.0.8.3

### CODE REVIEW

**Comment by Dream-Master** [comment]
@richardhendricks why you enable  the extra life. So far ik crash the game if you consume the heart when you not in Hard-core mode. On the other hand you not save the heart for the canister heart use later on.

File: config/betterquesting/DefaultQuests.json [hunk 1]
```diff
@@ -112792,12 +112792,6 @@
               "Count:3": 1,
               "Damage:2": 0,
               "OreDict:8": ""
-            },
```
Comment (Dream-Master) [inline config/betterquesting/DefaultQuests.json]: Wasn't this necessary for crafting or why this is removed?

File: config/GTNewHorizons/blocklimiter.cfg [hunk 2]
```diff
@@ -19,7 +19,6 @@ main {
     # Define your Items here. Syntax is: [modID]:[ItemID];[DimID];... if you don't add a Dimension (e.g. minecraft:dirt instead of minecraft:dirt;12) it will be denied in every dimension [default: ]
     S:ItemList <
         Avaritia:Endest_Pearl;0;-1;1;7;
-        betterquesting:extra_life;
         gregtech:gt.metaitem.01:32405;
```
[the remark this answers is not in the document]
Comment (richardhendricks) [reply config/GTNewHorizons/blocklimiter.cfg]: This is for the #5349 ticket.  I tried eating a Heart outside of Hardcore Questing mode, and it did not allow it in BQ3, so I think it is fine to remove it from Block Limiter. They are farmable, so I think it doesn't matter if a player uses some for HQM lives, they can just do Kill All The Things quests to get more.

File: config/betterquesting/DefaultQuests.json [hunk 3]
```diff
@@ -41,7 +41,7 @@
           "requiredItems:9": {
             "0:10": {
               "id:8": "minecraft:dirt",
-              "Count:3": 16,
+              "Count:3": 8,
               "Damage:2": 0,
```
[the remark this answers is not in the document]
Comment (richardhendricks) [reply config/betterquesting/DefaultQuests.json]: Eh, I just found it really annoying to have to get that much before you have any tools at all. I wanted to reduce the wood log amount too, but wood is more important.

File: config/GTNewHorizons/blocklimiter.cfg (hunk 2 above)
[the remark this answers is not in the document]
Comment (Dream-Master) [reply config/GTNewHorizons/blocklimiter.cfg]: ok

File: config/betterquesting/DefaultQuests.json (hunk 3 above)
[the remark this answers is not in the document]
Comment (Dream-Master) [reply config/betterquesting/DefaultQuests.json]: Yea it isnt that important

File: config/betterquesting/DefaultQuests.json [hunk 4]
```diff
@@ -112792,12 +112792,6 @@
               "Count:3": 1,
               "Damage:2": 0,
               "OreDict:8": ""
-            },
-            "3:10": {
-              "id:8": "minecraft:diamond_sword",
-              "Count:3": 1,
-              "Damage:2": 0,
-              "OreDict:8": ""
             }
```
Comment (Dream-Master) [inline config/betterquesting/DefaultQuests.json]: I mean this sword
why it is removed?

File: config/betterquesting/DefaultQuests.json (hunk 1 above)
Comment (richardhendricks) [reply config/betterquesting/DefaultQuests.json]: The diamond sword is not necessary for crafting, it was just a requirement for the quest based on "needing a good sword" but by this point a player can have many options for a sword. See #5233.

**Pull request merged**